mindee/parsing/standard/text.py

```diff
@@ -1,6 +1,7 @@
 from typing import Optional
 
-from mindee.parsing.standard.base import BaseField, FieldPositionMixin, TypePrediction
+from mindee.parsing.common.string_dict import StringDict
+from mindee.parsing.standard.base import BaseField, FieldPositionMixin
 
 
 class TextField(FieldPositionMixin, BaseField):
```

Comment:
Other libraries have this as `StringField`, no?

(I actually like `TextField` better, but more important to harmonize with others)

Reply:
Totally agree